pkg/cmd/get/secrets.go

```diff
@@ -46,8 +47,11 @@ var (
 )
 
 type TemplateData struct {
+	IsCore    bool              `json:"isCore"`
 	Name      string            `json:"name"`
 	Namespace string            `json:"namespace"`
+	Username  string            `json:"username"`
+	Password  string            `json:"password"`
 	Data      map[string]string `json:"data"`
 }
```

Comment:
Why are we duplicating fields from the secret struct? I thought this struct isn't used?

Reply:
We can use struct from entity package in that case.

Reply:
This struct isn't used anywhere now, correct? We should remove it.